apiary.apib

```diff
@@ -5,6 +5,195 @@ HOST: https://operationcode.org/
 
 API endpoints that Operation Code's Rails backend makes available to its React frontend.
 
+## CodeSchool | Collection [/api/v1/code_schools]
+
+### List All CodeSchool Members [GET]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "id": 112,
+                "name": "Wyncode Academy",
+                "url": "http://wyncode.co",
+                "logo": "https://raw.githubusercontent.com/OperationCode/operationcode_frontend/master/src/images/codeSchoolLogos/wyncode.jpg",
+                "full_time": true,
+                "hardware_included": false,
+                "has_online": false,
+                "online_only": false,
+                "created_at": "2017-09-21T05:48:50.056Z",
+                "updated_at": "2017-09-21T05:48:50.056Z",
+                "notes": null
+            },
+            {
+                "id": 113,
+                "name": "Nashville Software School",
+                "url": "http://nashvillesoftwareschool.com/",
+                "logo": "https://raw.githubusercontent.com/OperationCode/operationcode_frontend/master/src/images/codeSchoolLogos/nss.jpg",
+                "full_time": true,
+                "hardware_included": false,
+                "has_online": false,
+                "online_only": false,
+                "created_at": "2017-09-21T05:48:50.108Z",
+                "updated_at": "2017-09-21T05:48:50.108Z",
+                "notes": null
+            }
+        ]
+        
+        
+### CodeSchool | Create [/api/v1/code_schools{?name,url,logo,full_time,hardware_included,has_online,online_only,notes}]
```

Comment:
I believe we'll want this to be an `h2` header instead of an `h3`, with an `h3` beneath it, because it is its own endpoint.